back/spec/requests/google_authentication_spec.rb

```diff
@@ -111,15 +111,17 @@
     get "/auth/google?random-passthrough-param=somevalue"
     follow_redirect!
 
-    expect(response).to redirect_to("/en/complete-signup?random-passthrough-param=somevalue")
+    # Expect the redirect url to include the locale ('nl-NL') from Tenant locales that includes
+    # the locale code in the mock omniauth response ('nl')
+    expect(response).to redirect_to("/nl-NL/complete-signup?random-passthrough-param=somevalue")
```

Comment:
Now, we should be matching with the 'nl' locale returned in OmniAuth response, so 2 tests changed to reflect this.